CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 0.1.9
+Released on Januay 9, 2023.
+
+### Key Features
+* As of this release, Aqueduct has usage tracking. Usage tracking is fully
+    anonymized and captures API routes, performance data, and error rates
+    without revealing any specifics of your machine or deployment. For more
+    details, check out our [documentation](https://docs.aqueducthq.com/usage).
+* We now support cascading workflow triggers. You can specify that in our [python SDK](https://github.com/aqueducthq/aqueduct/blob/main/sdk/aqueduct/client.py#L373).
```

Comment:
@likawind maybe add a sentence explaining what it means. Sth like "We now support cascading workflow triggers, meaning that a workflow can trigger another one at the end of its execution."